content/docs/welcome-guide/setup/setup-from-github.md

```diff
@@ -5,7 +5,7 @@ weight: 300
 toc: true
 ---
 
-Getting the source for Open 3D Engine (O3DE) from GitHub is a great way to set up your development environment, so you can easily sync future engine updates and make contributions to the open source project base. Check out the video below fo a step by step guide to set up O3DE from GitHub, or follow the tutorial on this page.
+Getting the source for Open 3D Engine (O3DE) from GitHub is a great way to set up your development environment, so you can easily sync future engine updates and make contributions to the open source project base. Check out the video below for a step by step guide to set up O3DE from GitHub, or follow the tutorial on this page.
```

Comment:
```suggestion
Getting the source for Open 3D Engine (O3DE) from GitHub is a great way to set up your development environment. This method lets you easily sync future engine updates and make contributions to the open source project base. For instructions on setting up O3DE from GitHub, check out the following video or the tutorial on this page.
```
* For the first sentence:
    * It's a long one. I recommend splitting it in two for flow, readability.
    * Be cautious around using "a great way" and "easily". These can sound a bit like Marketing-speak.
* Re-ordered the last sentence to for flow, focus.